pkg/grpc/management/proto/management.proto

```diff
@@ -2022,8 +2022,8 @@ message UserMultiFactor {
 
 enum MfaType {
   MFATYPE_UNSPECIFIED = 0;
-  MFATYPE_SMS = 1;
-  MFATYPE_OTP = 2;
+  MFATYPE_OTP = 1;
```

Comment:
Does this have som einpact to console?

Reply:
i changed console/src/assets/i18n/de.json and console/src/assets/i18n/en.json
availability check is done by enum
and sms isn't used